content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,220 @@
+
+**custom check** - If you have a unique system that you want to monitor, or if you're going to expand the metrics already sent by an integration, you can build a [custom check][7] to define and send metrics to Datadog. However, if you want to monitor a generally available application, public service, or an open source project, and we don't have an integration that already exists for it, you should consider [building a new integration][1] instead of a custom check.
+
+**tagging** - [Tags][13] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][14] to manage and analyze them.
+
+**event** - Events are informational messages about your system that are consumed by the events stream.
+
+**service check** - Service checks are a type of monitor used to track the status of the service.
+
+## Setting up an integration
+
+The Datadog Agent package includes integrations officially supported by Datadog, in [integrations core][2]. To use the integrations in integrations core, download the Datadog agent. Community-based integrations are in [integrations extras][3], and to use those, you need to download them individually. For more information on installing or managing these integrations, see the [integrations management guide][15].
+
+### API and Application keys
+
+In order to [install the Datadog Agent][6], you need an [API and Application key][16]. You can manage your accounts API and Application keys in the [API Settings page][17] of the UI.
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][18] for specific instructions on how to connect. For other supported integrations, install the [Datadog Agent][19]. Most integrations are supported on our containerized agents: [Docker][20], and [Kubernetes][21]. After you've downloaded the Agent, go to the [Integrations page][18] section to find specific configuration instructions for individual integrations.
+
+### Configuring Agent integrations
+
+Configurations are specific to [individual integrations][18]. In the `conf.d` folder at the root of your Agent's configuration directory, there is a file named `<INTEGRATIONS>.d` in each integration folder, and a sample `conf.yaml.example` that lists all available configuration options. 
+
+To begin configuring, rename `conf.yaml.example` (in `<INTEGRATIONS>.d`) to `conf.yaml` to activate the integration. Then update the parameters inside the configuration file. Non-required parameters are commented out. All configuration files follow the format documented in the [parameters documentation][22].
+
+All configuration files have things in common:
+* Not required parameters are commented out of the example files.
```

Comment:
we should replace this bullet list with the quick features of the paramters documentation (the overall format of param_name - type - required/optional ...)

Reply:
We should also add here that some Agent configurations files includes a log collection section at the bottom that can be configured for log collection.

Reply:
this is the only one where I still think it's better to just link out to the list elsewhere (I did fix the broken link though)